DependencyInjection/SncRedisExtension.php

```diff
@@ -427,7 +433,43 @@ protected function loadSwiftMailer(array $config, ContainerBuilder $container)
     }
 
     /**
-     * Loads the profiler storage configuration.
+     * Load the correct serializer for Redis
+     *
+     * @param string $type
+     *
+     * @return string
+     * @throws InvalidConfigurationException
+     */
+    public function loadSerializationType($type)
+    {
+        // not using constants, because it would fail if phpredis extension is not enabled
+        $types = array(
+            "none" => 0, // \Redis::SERIALIZER_NONE,
+            "php" => 1,  // \Redis::SERIALIZER_PHP,
+            "igbinary" => 2 // \Redis::SERIALIZER_IGBINARY
+        );
```

Comment:
Given the dependency on these constants, why can it even be invoked if phpredis is not loaded?

Also single quotes   ;)

Reply:
Do you mean to say that we should use  this instead?

$types = array(
            'none' => 0, \Redis::SERIALIZER_NONE,
            'php' => 1,  \Redis::SERIALIZER_PHP,
            'igbinary' => 2 \Redis::SERIALIZER_IGBINARY
        );

Reply:
The comment says we're using the integers instead of the constants as they would fail if phpredis is not loaded, but in that case the serialization wouldn't work anyway, so how come we have to account for it?